lib/espi_dni/slack_rtm.ex

```diff
@@ -1,25 +1,43 @@
 defmodule EspiDni.SlackRtm do
+
+  @moduledoc """
+  A module for dealing with slacks Realtime Messaging API
+  Uses https://github.com/BlakeWilliams/Elixir-Slack
+  """
+
   use Slack
   require Logger
 
   @token Application.get_env(:espi_dni, __MODULE__)[:slack_token]
 
   def start_link, do: start_link(@token)
 
+  @doc"""
```

Comment:
Missing space between `@doc` and `"""`, also at several other places in this file.